ide/static/ide/js/compile.js

```diff
@@ -516,6 +516,38 @@ CloudPebble.Compile = (function() {
         });
     };
 
+    var show_clear_logs_prompt = function() {
+        CloudPebble.Prompts.Confirm(gettext("Clear all app logs?"), gettext("This cannot be undone."), function() {
+            mLogHolder.empty();
+            mPreviousDisplayLogs = [];
+            mArchivePoints = [];
+            ga('send', 'event', 'logs', 'delete');
```

Comment:
As far as I can tell, keen is a server side thing only? This doesn't interact with the server.